solr/solrj/src/java/org/apache/solr/common/params/SolrParams.java

```diff
@@ -526,4 +526,9 @@ public String toString() {
     }
     return sb.toString();
   }
+
+  /** An empty, immutable SolrParams. */
+  public static SolrParams of() {
```

Comment:
[0] When I see `of()` it reminds me of the various var-arg methods with the same name that Java's "Collections" classes offer ([List.of](https://docs.oracle.com/javase/9/docs/api/java/util/List.html#of--)).

It's a little jarring to see that we don't accept any args here, and that this is only for creating empty param-sets.  Maybe a name like `emptyParams()` or `empty()` would convey things more accurately?

Reply:
Coincidentally I recently had the same conversation for the new SimpleOrderedMap.of.    It's modelled off JDK data structure utilities using this name, thus its consistent.  Once we see additional overloads, I think you'll like it better.  I could add one or two in this PR.